lib/http-server.js

```diff
@@ -107,6 +107,14 @@ function HttpServer(options) {
 
   if (typeof options.proxy === 'string') {
     var proxy = httpProxy.createProxyServer({});
+    proxy.on('error', function (err) {
+      if (options.logFn) { // If there's a log function for requests we can log
```

Comment:
If there's a log function set, the code should use that and not just `console.info`. See https://github.com/indexzero/http-server/blob/94329688b5bf6c058a9f2c24739bd628b7345dce/lib/http-server.js#L62-L64

Reply:
Should I fake `req` and `res` arguments in this case? I don't have them in this context

Reply:
ping

Reply:
Move the `on('error')` code into the `before.push` https://github.com/indexzero/http-server/blob/f65f1a7154456f823261c4f8e90b35eff66eb23a/lib/http-server.js#L118 where the proxy is added into the mix.

Reply:
Found a cleaner way to do it. Force-pushed, please review 🙇